crawler/__init__.py

```diff
@@ -33,8 +32,11 @@ def create_app(config_object: str = None) -> flask.Flask:
         scheduler.start()
 
     config, _ = get_config(config_object or "")
+
```

Comment:
In `cherrypicker_test_data`, you cast the result of `get_config` to a Tuple, but you don't here - not sure if that's an important difference or not

Reply:
mypy was not complaining with this one because config does not have a type associated in this code